File: src/com/fusenetworks/fuse/listener/CommandBlocker.java
package com.fusenetworks.fuse.listener;
 
import static com.fusenetworks.fuse.Fuse.server;
import org.bukkit.plugin.PluginManager;
import org.bukkit.ChatColor;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerCommandPreprocessEvent;
import org.bukkit.event.EventHandler;

public class CommandBlocker implements Listener {
    final PluginManager pm = server.getPluginManager();
    @EventHandler
    public boolean PlayerCommandPreprocessEvent(PlayerCommandPreprocessEvent event) {
    String command = event.getMessage();
    final String[] commandParts = command.split(" ");
        if (commandParts[0].contains(":"))
        {
            event.getPlayer().sendMessage(ChatColor.GRAY + "Plugin specific commands are disabled");
            event.setCancelled(true);
            return true;
        }
    return true;
    }
}

File: src/com/fusenetworks/fuse/listener/CommandBlocker.kt
package com.fusenetworks.fuse.listener

import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerCommandPreprocessEvent

class CommandBlocker : Listener {

    val pm = Bukkit.getServer().getPluginManager()

    @EventHandler
    fun PlayerCommandPreprocessEvent(event: PlayerCommandPreprocessEvent): Boolean {
        val command = event.message
        val commandParts = command.split(" ".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        if (commandParts[0].contains(":")) {
            event.player.sendMessage(ChatColor.GRAY.toString() + "Plugin specific commands are disabled")
            event.isCancelled = true
            return true
        }
        return true
    }
}